src/main/java/org/spburegistry/backend/service/implementation/UserServiceImpl.java

```diff
@@ -0,0 +1,34 @@
+package org.spburegistry.backend.service.implementation;
+
+import org.spburegistry.backend.entity.User;
+import org.spburegistry.backend.repository.UserRepo;
+import org.spburegistry.backend.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class UserServiceImpl implements UserService {
+
+    private final UserRepo userRepo;
+
+    @Autowired
+    public UserServiceImpl(UserRepo userRepo) {
+        this.userRepo = userRepo;
+    }
+
+    @Override
+    public User findById(long id) {
+        return userRepo.findById(id).get();
```

Comment:
`Optional#get()` is not a good practice. Please, handle null value and return response with error.

Reply:
I think, it is better to add ExceptionHandler to start work properly with these types of exception.